.eslintrc

```diff
@@ -72,7 +72,8 @@
         "chatbubbles", "nonexisting", "primarytext", "autocomple", "stringify", "backend",
         "Menlo", "ok", "py", "todo", "Mc", "lodash", "selectable", "isequal", "lightgray", "tc",
         "zulipchat", "prepend", "pierre", "allen", "jan", "donald", "jane", "unicode", "joe",
-        "dan", "abramov", "lang", "Crashlytics", "bool", "ionicons", "truthy", "bezier", "decrement", "js", "jsonp",
+        "dan", "abramov", "lang", "Crashlytics", "bool", "ionicons", "truthy", "bezier", "decrement",
+        "jsonp", "js", "actionsheet", "ionicons", "Un", "Unmute"
```

Comment:
You will not need the 'Un' here if you rename everything from 'unMute' to 'unmute' (which is more correct)